blueprintapi/urls.py

```diff
@@ -34,6 +35,7 @@
 urlpatterns = [
     path("", views.index, name="index"),
     path("admin/", admin.site.urls),
+    path('api-token-auth/', views.UserObtainTokenView.as_view()),
```

Comment:
Was it intentional to use this url instead of conforming to the others by starting with api/?